command/commands/user.py

```diff
@@ -218,12 +218,22 @@ def view_helper(self, user_id, slack_id):
         except LookupError:
             return self.lookup_error, 200
 
-    def add_helper(self, user_id):
+    def add_helper(self, user_id, use_force):
         """
         Add the user to the database via user id.
 
         :param user_id: Slack ID of user to be added
+        :param use_force: If this is set, we store the user even if they are
+        already added in the database
         :return: ``"User added!", 200``
         """
+        # Try to look up and avoid overwriting if we are not using force
+        if not use_force:
+            try:
+                self.facade.retrieve_user(user_id)
```

Comment:
It might be a good idea to return a message here, maybe something along the lines of "that user already exists; to overwrite with a new user use -f".